kafka/adminapi.go

```diff
@@ -1505,6 +1528,22 @@ func (a *AdminClient) cConfigResourceToResult(cRes **C.rd_kafka_ConfigResource_t
 	return result, nil
 }
 
+// cToDeletedRecordResult converts a C topic partitions list to a Go DeleteRecordsResult slice.
+func cToDeletedRecordResult(cparts *C.rd_kafka_topic_partition_list_t) (results []DeleteRecordsResult) {
+	partcnt := int(cparts.cnt)
+	results = make([]DeleteRecordsResult, partcnt)
+	partitions := newTopicPartitionsFromCparts(cparts)
+
+	for i := 0; i < partcnt; i++ {
+		results[i].TopicPartition = partitions[i]
+		if results[i].TopicPartition.Error == nil {
+			results[i].DeletedRecords = &DeletedRecords{LowWatermark: results[i].TopicPartition.Offset}
```

Comment:
Let's break this and other long lines
```suggestion
			results[i].DeletedRecords = &DeletedRecords{
				LowWatermark: results[i].TopicPartition.Offset}
```